docs/_scripts/link_map.py

```diff
@@ -67,58 +67,57 @@
 JS_LINK_MAP = {
```

Comment:
@hntrl can you review these? the commented out ones i couldn't find and i'm not sure if that's because they were deprecated or if i just missed them